package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.0.0-semantically-released",
   "description": "Simple and complete custom Cypress commands and utilities that encourage good testing practices.",
   "main": "dist/index.js",
-  "typings": "typings",
+  "typings": "dist/typings",
```

Comment:
Instead of moving the files, could we just include `typings` in the `files` property here?

Reply:
That's what I tried first but for some reason it wasn't working. I assume it's because of the way `kcd-scripts build` works. It seems to take files from src only.